Source/WebKit/UIProcess/API/APIFeature.cpp

```diff
@@ -28,16 +28,17 @@
 
 namespace API {
 
-Ref<Feature> Feature::create(const WTF::String& name, const WTF::String& key, FeatureStatus status, const WTF::String& details, bool defaultValue, bool hidden)
+Ref<Feature> Feature::create(const WTF::String& name, const WTF::String& key, FeatureStatus status, FeatureCategory category, const WTF::String& details, bool defaultValue, bool hidden)
 {
-    return adoptRef(*new Feature(name, key, status, details, defaultValue, hidden));
+    return adoptRef(*new Feature(name, key, status, category, details, defaultValue, hidden));
 }
 
-Feature::Feature(const WTF::String& name, const WTF::String& key, FeatureStatus status, const WTF::String& details, bool defaultValue, bool hidden)
+Feature::Feature(const WTF::String& name, const WTF::String& key, FeatureStatus status, FeatureCategory category, const WTF::String& details, bool defaultValue, bool hidden)
     : m_name(name)
     , m_key(key)
-    , m_status(status)
```

Comment:
nit: Are these in the right order? I see that they don’t match the order of the constructor parameters.

Reply:
These are in the correct initialization order to avoid a compile error. I adjusted the order slightly to group the small date types to improve packing.